stubs/Symfony/Component/Form/DataMapperInterface.stub

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Symfony\Component\Form;
+
+/**
+ * @template TViewData
+ */
+interface DataMapperInterface
+{
+    /**
+     * @param TViewData|null $viewData
+     * @param \Traversable<mixed, FormInterface<mixed>> $forms
+     */
+    public function mapDataToForms(mixed $viewData, \Traversable $forms): void;
+
+    /**
+     * @param \Traversable<mixed, FormInterface<mixed>> $forms
+     * @param TViewData|null $viewData
+     * @param-out TViewData $viewData
```

Comment:
This is wrong. there is no guarantee that the reference argument is replaced by a non-null value by the end of the method (and actually, the 3 core implementations of the interface have cases where the data will be `null` in the output)